stdlib/LinearAlgebra/src/bunchkaufman.jl

```diff
@@ -62,6 +66,11 @@ as appropriate given `S.uplo`, and `S.p`.
 If `rook` is `true`, rook pivoting is used. If `rook` is false,
 rook pivoting is not used.
 
+When `check = true` the matrix is checked for singularity, and makes sure
+the factorization is successful, or else throw an error. When `check = false`
```

Comment:
Perhaps slightly more compact "When `check = true`, an error is thrown if the decomposition fails. When `check = false`, responsibility for checking the decomposition's validity (via [`issuccess`](@ref)) lies with the user." ?